### DISCUSSION TITLE
f88e2745-24a0-4922-9bbe-ef831fd7fa64

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9568] [Rules Discussions]
# f88e2745-24a0-4922-9bbe-ef831fd7fa64

Do you send "As Per Our Conversation" emails to ensure important conversations are documented and decisions are confirmed?

https://www.ssw.com.au/rules/as-per-our-conversation-emails/

**Comment by Calinator444** [comment]
For the heading "Location - include the reason at the top" don't the examples mostly apply when making it clear that you are sending a task? I agree that getting the subject of the conversation across first and foremost in an email is important but I don't think this topic specifically relates to as per our conversation emails.

**Comment by designbyalex** [comment]
I learnt the gold standards of emails, drafting the email, sharing your screen and getting it checked. 

Also, how to reference any verbal communication using "as per our conversation."

**Comment by ClaraFang-SSW** [comment]
As per our conversation" is a really practical rule, it makes sure everyone's on the same page and stops people from going back and forth endlessly. I used this with my parents, as we kept going in circles on our renovation. 🤣
Btw, loved the video, young Uly and Adam!
